launch/cron-admin.yml

```diff
@@ -7,7 +7,7 @@ env:
 - PORT
 resources:
   cpu: 0.1
-  memory: 100
```

Comment:
whoaz -- i guess this hadn't been deployed since this was changed from mb to gb 🤔